src/key.js

```diff
@@ -359,12 +385,22 @@ class Key {
   /**
    * Concats one or more Keys into one new Key.
    *
-   * @param {Array<Key>} keys
+   * @param {Array<Key>} keys - The array of keys to concatenate
    * @returns {Key}
    */
   concat (...keys) {
     return Key.withNamespaces([...this.namespaces(), ...flatten(keys.map(key => key.namespaces()))])
   }
+
+  /**
+   * Check if value is a Key instance
+   *
+   * @param {any} value - Value to check
+   * @returns {boolean}
+   */
+  static isKey (value) {
```

Comment:
I think we shouldn't do this sort of thing any more and just use `instanceof`.  `Boolean(value && value[symbol])` doesn't make any guarantees that what's being tested is compatible with the caller's expectation of the API.

It will break when two copies of this module are in the dep tree, but I think that's better than the returned type being incompatible.